src/sign/eip1271.ts

```diff
@@ -48,8 +48,8 @@ export async function verifyOldVersion(
   return returnValue.toLowerCase() === magicValue.toLowerCase();
 }
 
-export async function verify(address, sig, hash, network = '1') {
-  const provider = getProvider(network);
+export async function verify(address, sig, hash, network = '1', opts) {
+  const provider = getProvider(network, opts);
```

Comment:
It is not optional? then we should change it in sequencer too? 🤔

Reply:
it's optional